README.md

```diff
@@ -198,3 +198,130 @@ L'applicazione utilizza un custom handler per intercettare e notificare errori j
+
+Un esempio più complesso permette di utilizzare le funzioni avanzate del layer di theming di native-base.
+
+```javascript
+import { type Theme } from '../types'
+import variables from '../variables'
+
+export default (): Theme => {
+  const theme = {
+    '.spacer': {
+      '.large': {
+        height: variables.spacerLargeHeight
+      },
+
+      height: variables.spacerHeight
+    },
+
+    '.footer': {
+      paddingTop: variables.footerPaddingTop,
+      paddingLeft: variables.footerPaddingLeft,
+      paddingBottom: variables.footerPaddingBottom,
+      paddingRight: variables.footerPaddingRight,
+      backgroundColor: variables.footerBackground,
+      borderTopWidth: variables.footerShadowWidth,
+      borderColor: variables.footerShadowColor
+    }
+  }
+
+  return theme
+}
+```
+
+All'interno del file del tema di un singolo componente è possibile infatti definire degli attributi specifici che verranno utilizzati sono nel caso il componente in questione abbia una specifica prop.
+Definendo nell'oggetto di tema qualcosa come
+```javascript
+'.footer': {
+      paddingTop: variables.footerPaddingTop
+    }
+```
+sarà possibile quando necessario utilizzare il componente associandogli la property `footer` nel seguente modo `<Component footer />` ed automaticamente il sistema di theming applicherà al componente gli attributi definiti (`paddingTop: variables.footerPaddingTop`).
+
+Altra funzione avanzata è quella che permette di definire il tema dei componenti figli a partire dal componente padre.
+Prediamo come esempio il seguente frammento di codice di un generico componente:
+
+```javascript
+...
+render() {
+  return(
+    <Content>
+      <Button>
+        <Text>My button</Text>
+      </Button>
+    </Content>
+  )
+}
+...
+```
+
+La libreria native-base permette di definire l'aspetto del componente figlio `Text` presente all'interno del componente padre `Button`. Ad esempio per definire la dimensione del testo all'interno di tutti i bottoni presenti nell'applicazione è possibile all'interno del file `/js/theme/components/Button.js` il seguente codice:
+
+```javasscript
+import variables from '../variables'
+
+export default (): Theme => {
+  const theme = {
+    'NativeBase.Text': {
+      fontSize: variables.btnTextFontSize
+    }
+  }
+
+  return theme
+}
+```
+
+è possibile spingersi ancora oltre e combinare le due funzionalità viste in precedenza.
+
+```javasscript
+import variables from '../variables'
+
+export default (): Theme => {
+  const theme = {
+    '.small': {
+      'NativeBase.Text': {
+        fontSize: variables.btnTextFontSize
+      }
+    }
+  }
+
+  return theme
+}
+```
+
+I questo caso quanto definito per all'interno dell'attributo `NativeBase.Text` sarà utilizzato solo nel caso il bottone abbia associata una props dal nome `small`.
```

Comment:
In questo, caso quanto definito all'interno dell'attributo `NativeBase.Text` sarà utilizzato solo nel caso in cui il bottone abbia associata una proprietà dal nome `small`.